<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserPage</class>
 <widget class="QMainWindow" name="UserPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget{
	border-image: url(:/image/userPage/background.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1280</width>
     <height>720</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>681</width>
      <height>338</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="1" column="0">
        <widget class="QPushButton" name="delButton">
         <property name="font">
          <font>
           <family>Nazanin</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">#delButton{
border-radius: 10px;
background-color: #ae9700;
border: none;
color: #FFFFFF;
text-align: center;
padding: 10px;
margin: 5px;
}
#delButton:hover {
  background-color: #5f2630;
}
</string>
         </property>
         <property name="text">
          <string>حذف</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="delAllButton">
         <property name="font">
          <font>
           <family>Nazanin</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">#delAllButton{
border-radius: 10px;
background-color: #ae9700;
border: none;
color: #FFFFFF;
text-align: center;
padding: 10px;
margin: 5px;
}
#delAllButton:hover {
  background-color: #5f2630;
}
</string>
         </property>
         <property name="text">
          <string>حذف همه</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QPushButton" name="resButton">
         <property name="font">
          <font>
           <family>Nazanin</family>
           <pointsize>13</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">#resButton{
border-radius: 10px;
background-color: #ae9700;
border: none;
color: #FFFFFF;
text-align: center;
padding: 10px;
margin: 5px;
}
#resButton:hover {
  background-color: #5f2630;
}
</string>
         </property>
         <property name="text">
          <string>رزرو</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QPushButton" name="addButton">
         <property name="font">
          <font>
           <family>Nazanin</family>
           <pointsize>11</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">#addButton{
border-radius: 10px;
background-color: #ae9700;
border: none;
color: #FFFFFF;
text-align: center;
padding: 10px;
margin: 5px;
}
#addButton:hover {
  background-color: #5f2630;
}
</string>
         </property>
         <property name="text">
          <string>اضافه</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" rowspan="3">
        <widget class="QListWidget" name="listWidget">
         <property name="font">
          <font>
           <family>Nazanin</family>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="toolTip">
          <string/>
         </property>
         <property name="layoutDirection">
          <enum>Qt::RightToLeft</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="spacing">
          <number>2</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>30</y>
      <width>561</width>
      <height>661</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nazanin</family>
      <pointsize>11</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Search">
     <attribute name="title">
      <string>جستجو</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>531</width>
        <height>521</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLineEdit" name="Serach_lineEdit">
         <property name="font">
          <font>
           <family>Nazanin</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">
padding : 10px;
background-color: transparent;
border-radius: 15px;
border: 3px solid #ae9700;
</string>
         </property>
         <property name="placeholderText">
          <string>جستجو ...</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="font">
          <font>
           <family>Nazanin</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="title">
          <string>: بر اساس</string>
         </property>
         <widget class="QWidget" name="layoutWidget">
          <property name="geometry">
           <rect>
            <x>100</x>
            <y>60</y>
            <width>411</width>
            <height>341</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QRadioButton" name="Name_RButton">
             <property name="font">
              <font>
               <family>Nazanin</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::RightToLeft</enum>
             </property>
             <property name="text">
              <string>نام فیلم</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Genre_RButton">
             <property name="font">
              <font>
               <family>Nazanin</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>ژانر</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="Dir_RButton">
             <property name="font">
              <font>
               <family>Nazanin</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>کارگردان</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QRadioButton" name="ReleaseDate_RButton">
             <property name="font">
              <font>
               <family>Nazanin</family>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>سال تولید</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="Reserved_Films">
     <attribute name="title">
      <string>فیلم های رزرو شده</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>511</width>
        <height>240</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Nazanin</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>فیلم های رزرو شده</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QListWidget" name="Reserve_listWidget">
         <property name="font">
          <font>
           <family>Nazanin</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="User">
     <attribute name="title">
      <string>ناحیه کاربری</string>
     </attribute>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>501</width>
        <height>105</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="namkarbari">
           <property name="font">
            <font>
             <family>Nazanin</family>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>نام کاربری :</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="username">
           <property name="font">
            <font>
             <family>Microsoft Himalaya</family>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="tedad_film">
           <property name="font">
            <font>
             <family>Nazanin</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>تعداد فیلم های رزرو شده :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="count_reserved">
           <property name="font">
            <font>
             <family>Microsoft Himalaya</family>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <family>Nazanin</family>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>رمز عبور :</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="password">
           <property name="font">
            <font>
             <family>Microsoft Himalaya</family>
             <pointsize>17</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>0</y>
      <width>75</width>
      <height>29</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>7</number>
     </property>
     <item>
      <widget class="QPushButton" name="minBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="MySrc.qrc">
         <normaloff>:/image/userPage/minimize.png</normaloff>:/image/userPage/minimize.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="MySrc.qrc">
         <normaloff>:/image/welcomePage/close.png</normaloff>:/image/welcomePage/close.png</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>681</width>
      <height>351</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <property name="font">
        <font>
         <family>Nazanin</family>
         <pointsize>11</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <item>
        <property name="text">
         <string>همه ژانر ها</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectItems</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideRight</enum>
       </property>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>نام فیلم</string>
        </property>
        <property name="font">
         <font>
          <family>Nazanin</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ژانر</string>
        </property>
        <property name="font">
         <font>
          <family>Nazanin</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>کارگردان</string>
        </property>
        <property name="font">
         <font>
          <family>Nazanin</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>سال تولید</string>
        </property>
        <property name="font">
         <font>
          <family>Nazanin</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>ظرفیت باقیمانده</string>
        </property>
        <property name="font">
         <font>
          <family>Nazanin</family>
          <pointsize>11</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="LogOutBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>33</width>
      <height>27</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="MySrc.qrc">
      <normaloff>:/image/userPage/LogOut.png</normaloff>:/image/userPage/LogOut.png</iconset>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="MySrc.qrc"/>
 </resources>
 <connections/>
</ui>
